package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@xlayers/vscode-extension",
+  "name": "xlayers-extension",
```

Comment:
What's the reason for this change?

Reply:
We can't use scoped names here. We can use any name as long it is not scoped. Do you have a suggestion for the name?